roadmap.md

```diff
@@ -90,3 +90,10 @@ This scenario loads a web page and prints it as a PDF. In spec terms, this invol
 
 - [x] Some items from the previous milestones
 - [x] [Printing to PDF as Base64-encoded string](https://w3c.github.io/webdriver-bidi/#command-browsingContext-print)
+
+### Window management
+
+This scenario changes the browser window dimensions and position, maximizes/minimizes the window, and toggles full-screen mode. 
```

Comment:
One thing we tried to do for the initial scenarios is explain why a user might want this. Maybe something like:

> This scenario loads a web page with a responsive UI, …

E.g. how would a web page respond to the window being maximized/minimized?

Reply:
I am not sure it will be feasible to list all possible side effects but I assume the general use case is 1) for testing web observable side effects of this 2) for testing the browser alongside other desktop apps

Reply:
We don't need it for Puppeteer but it seems high on the list of other tools.

Reply:
I guess the user scenarios are:
* Ensuring the page responds as expected when the viewport dimensions change
* Verifying that the page works as expected when the document is hidden (minimize window is the easiest way to ensure this in platforms that support it)
* Ensuring that the page works as expected in fullscreen mode (especially given that there's an API for that, and pages can have specific fullscreen behaviour).

Reply:
> Ensuring the page responds as expected when the viewport dimensions change

browsingContext.set_viewport can be used for this.

> Verifying that the page works as expected when the document is hidden (minimize window is the easiest way to ensure this in platforms that support it)

browsingContext.activate would be the cross platform way.

At the same time implementing only the fullscreen toggle seems very limiting.

Reply:
> browsingContext.activate would be the cross platform way.

Not really; if you have two browsing contexts they might both be visible at the same time (e.g. because they're in different windows rather than different tabs), so there's no guarantee that activating one context makes the other hidden. But, on platforms that support it, minimize does pretty much guarantee that any contexts in that (OS) window are hidden.

Reply:
> Not really; if you have two browsing contexts they might both be visible at the same time (e.g. because they're in different windows rather than different tabs), so there's no guarantee that activating one context makes the other hidden. But, on platforms that support it, minimize does pretty much guarantee that any contexts in that (OS) window are hidden.

is there a real-world use case for testing that two contexts visible at the same time in different windows? my comment was for testing the visibility change in isolation.